src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -392,6 +402,43 @@ class HaskellCompletionContributor extends CompletionContributor {
     waitForResult(lookupElements)
   }
 
+  private def getIdsFromAllExportedModules(project: Project) = {
```

Comment:
OK, so after loading current file in REPL, `BrowseModuleComponent` will kindly work as expected for me right? 😄